app/components/dashboard/about/QuotesExhibition.js

```diff
@@ -0,0 +1,106 @@
+export default function QuotesExhibition() {
```

Comment:
component is too big

component as a lot of repetition

break it up

Reply:
Done! The component was simplified